config/v3_0_experimental/types/schema.go

```diff
@@ -96,7 +96,7 @@ type LinkEmbedded1 struct {
 
 type Node struct {
 	Group     *NodeGroup `json:"group,omitempty"`
-	Overwrite *bool      `json:"overwrite,omitempty"`
+	Overwrite bool       `json:"overwrite,omitempty"`
```

Comment:
We'll want this to be a pointer eventually for the config merging. This is to allow a child config to overwrite a parent config if the child sets it to false and the parent sets it to true.

It's not a problem for this PR though since there are a _lot_ of fields that will need to be changed and we can do it then, just an FYI.

Reply:
Makes sense, thanks for the heads up. Sounds good to leave it for this PR and change it back to a pointer later, while the other fields are being reworked.